### DISCUSSION TITLE
Object-Based Permissions: Can not select objects type that do not follow each other

### DISCUSSION THREAD

**Discussion by peppermintea** [netbox-community/netbox#10340] [Q&A] [answered]
Hi!

netbox version: 3.3.2

How can I select mutiple objects type that do not follow each other ?

example: 
To adding mutiple object types to the same  permission, I can select :
DCIM>region
DCIM>site
DCIM>site group

But I can't select:
DCIM>location
DCIM>region
DCIM>site
DCIM>site group

![netbox_permissions](https://user-images.githubusercontent.com/75941610/189701438-cc60058d-fc70-4fa9-a4a0-78baf426d1c4.PNG)

**Comment by jeremystretch** [comment, marked as answer]
You can hold down the control key to select multiple non-consecutive items in a multi-selection list.

**Reply by peppermintea** [reply]
Thanks a lot!
